modules/rostests/apitests/ntdll/RtlGetNtProductType.c

```diff
@@ -153,15 +153,15 @@ START_TEST(RtlGetNtProductType)
     /* Change product type in the registry */
     ok_char(ChangeNtProductType(NtProductWinNt), TRUE);
     ok_char(RtlGetNtProductType(&ProductType2), TRUE);
-    ok_long(ProductType2, NtProductWinNt);
+    ok_long(ProductType2, ProductType);
 
     ok_char(ChangeNtProductType(NtProductLanManNt), TRUE);
     ok_char(RtlGetNtProductType(&ProductType2), TRUE);
-    ok_long(ProductType2, NtProductLanManNt);
+    ok_long(ProductType2, ProductType);
 
     ok_char(ChangeNtProductType(NtProductServer), TRUE);
     ok_char(RtlGetNtProductType(&ProductType2), TRUE);
-    ok_long(ProductType2, NtProductServer);
+    ok_long(ProductType2, ProductType);
```

Comment:
Aha, so indeed, we observe that on Windows, the product type does *not* change! That was exactly my comment in the other PR:
https://github.com/reactos/reactos/pull/3258#issuecomment-1137497172